code/datums/gameticker.dm

```diff
@@ -684,6 +685,26 @@ var/global/current_state = GAME_STATE_WORLD_INIT
 			if (it_is_ass_day)
 				earnings *= 2
 
+			//pilot's bonus check and reward
+			var/pilot_bonus = 500 //for receipt
+			if(in_centcom(player) && !isdead(player))
+				if (player.buckled)
+					if (istype(player.buckled,/obj/stool/chair/comfy/shuttle/pilot))
+						bank_earnings.pilot = 1
+						earnings += pilot_bonus
+				else if (isAI(player))
+					var/mob/living/silicon/ai/M = null
+					if (isAIeye(player))
+						M = player:mainframe
+					else
+						M = player
+					for (var/i in M.loc.contents)
+						if (istype(i,/obj/stool/chair/comfy/shuttle/pilot))
```

Comment:
Coul be replaced by something like `var/obj/stool/chair/comfy/shuttle/pilot/i = locate() in src.loc`.

Reply:
done